---
robot_params:
    - joint: shoulder_pan
      position_upper: 3.14159
      position_lower: -3.14159
      velocity: 1.57
      acceleration: 1.1
      envelope_position_radius:
          - [0.0, 0.0, 0.0, 80.0]

    - joint: shoulder_lift
      position_upper: 0.0
      position_lower: -3.14159
      velocity: 1.57
      acceleration: 1.5
      envelope_position_radius:
          - [0.0, 0.0, 0.0, 76.0]
          - [110.0, 0.0, 0.0, 60.0]
          - [210.0, 0.0, 0.0, 56.0]
          - [310.0, 0.0, 0.0, 56.0]
          - [420.0, 0.0, 0.0, 74.0]

    - joint: elbow
      position_upper: 1.57
      position_lower: -1.57
      velocity: 1.57
      acceleration: 1.8
      envelope_position_radius:
          - [10.0, 0.0, -115.0, 70.0]
          - [110.0, 0.0, -134.0, 55.0]
          - [210.0, 0.0, -134.0, 55.0]
          - [309.0, 0.0, -134.0, 55.0]
          - [385.0, 0.0, -134.0, 65.0]

    - joint: wrist_1_link
      position_upper: 3.14159
      position_lower: -3.14159
      velocity: 1.57
      acceleration: 2.7
      envelope_position_radius:
          - [0.0, 6.0, -10.0, 55.0]

    - joint: wrist_2_link
      type: revolute
      position_upper: 3.14159
      position_lower: -3.14159
      velocity: 1.57
      acceleration: 2.7
      envelope_position_radius:
          - [0.0, -4.0, -10.0, 60.0]

    - joint: wrist_3_link
      position_upper: 3.14159
      position_lower: -3.14159
      velocity: 1.8
      acceleration: 2.0
      envelope_position_radius:
          - [0.0, 0.0, 0.0, 40.0]
